script/functions.sh

```diff
@@ -63,6 +63,11 @@ function run_cukes {
       RUBYOPT="-I${PWD}/../bundle -rbundler/setup" \
          PATH="${PWD}/bin:$PATH" \
          bin/cucumber --strict
+    elif is_mri_2; then
+      RUBYOPT="${RUBYOPT} -I${PWD}/../bundle -rbundler/setup" \
+         PATH="${PWD}/bin:$PATH" \
+         bin/cucumber --strict --format pretty features/configuring_responses/wrapping_the_original_implementation.feature:80
```

Comment:
I love how `--format pretty` immediately showed the underlying failure message!

Reply:
It actually didn't I had to add a false statement to get the spec output